shopping-order-service/src/main/java/shopping/order/service/OrderServiceImpl.java

```diff
@@ -29,6 +29,20 @@ public OrderResponse createOrder(OrderRequest orderRequest) {
 		return toOrderResponse(order);
 	}
 
+	@Override
+	public OrderResponse getOrderById(String productId) {
+		return toOrderResponse(repository.findById(productId).orElseThrow(() -> new RuntimeException("Not fount order "+productId)));
```

Comment:
You should add a custom ExceptionHandler implementation to handle the exceptions thrown while processing the requests